src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj

```diff
@@ -3,7 +3,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <NoWarn>$(NoWarn);CS1573</NoWarn>
     <DefineConstants>$(DefineConstants);REGISTRY_ASSEMBLY</DefineConstants>
-    <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
+    <IsPartialFacadeAssemblyOnNetfx>true</IsPartialFacadeAssemblyOnNetfx>
```

Comment:
I don't like that we need to introduce an indirection property because of this. As we control the TargetFramework property in the project file I would instead do 

```
<IsPartialFacadeAssembly Condition="$(TargetFramework.StartsWith('net4'))">true</IsPartialFacadeAssembly>
```

Reply:
Also IMO  the goal should be to remove the `TargetNet...` properties as $(TargetFrameworkIdentifier) should be used in such cases.

Reply:
So doing this won't work in the designer, since it adds TargetFramework to the end of the first propertygroup.  Perhaps we could force folks to create a second PropertyGroup and then add some project build validation to catch that.  Do we want to go that route?

Reply:
> TargetFrameworkIdentifier cannot be used within a PropertyGroup in the body of a project as it is not yet set until after the SDK's targets are evaluated.

I was referring to Anirudh's usage of TargetsNetFx in Directory.Build.targets where TargetFrameworkIdentifier should be available.

> So doing this won't work for design time build, since it adds TargetFramework to the end of the first propertygroup. Perhaps we could force folks to create a second PropertyGroup and then add some project build validation to catch that. Do we want to go that route?

Why is TargetFramework added at the end of the first propertygroup and not at the beginning? Can we fix that? Anyways, I would be fine with that. Sounds like a better approach than the indirection.

Reply:
That’s the behavior of MSBuild’s Project.SetProperty API. You can’t change it without breaking. We have asked both VS ProjectSystem and OmniSharp to switch to SetGlobalProperty so that design time build will be more consistent with actual build.

Reply:
> That’s the behavior of MSBuild’s Project.SetProperty API. You can’t change it without breaking. We have asked both VS ProjectSystem and OmniSharp to switch to SetGlobalProperty so that design time build will be more consistent with actual build.

Got it. Is there an issue that tracks the ask?

> One thing we could do is set TargetFramework in addition to setting TargetFrameworks.

What would we set that to?

Reply:
I believe @anipik has issues for both of those.

 > What would we set that to?

Nevermind this was a bad suggestion as it fixes design time build at the cost of breaking other scenarios. This was just something I noticed previously when testing the vs code behavior.

Reply:
https://github.com/OmniSharp/omnisharp-roslyn/issues/1738
https://github.com/dotnet/project-system/issues/5996